frontend/packages/dev-console/src/components/helm/HelmInstallPage.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import * as yup from 'yup';
+import { RouteComponentProps } from 'react-router-dom';
+import { Helmet } from 'react-helmet';
+import { PageHeading, history } from '@console/internal/components/utils';
+import { coFetchJSON } from '@console/internal/co-fetch';
+import { Formik } from 'formik';
+import NamespacedPage, { NamespacedPageVariants } from '../NamespacedPage';
+import { nameValidationSchema } from '../import/validation-schema';
+import HelmInstallForm from './HelmInstallForm';
+
+export type HelmInstallPageProps = RouteComponentProps<{ ns?: string }>;
+
+export const HelmInstallPage: React.FunctionComponent<HelmInstallPageProps> = ({ location }) => {
+  const searchParams = new URLSearchParams(location.search);
+  const chartURL = searchParams.get('chartURL');
+  const preselectedNamespace = searchParams.get('preselected-ns');
+
+  const initialValues = {
+    releaseName: '',
+  };
+
+  const validationSchema = yup.object().shape({
+    releaseName: nameValidationSchema,
+  });
+
+  const handleSubmit = (values, actions) => {
+    const { releaseName } = values;
+    coFetchJSON
+      .post(
+        `/api/console/helm/install?ns=${preselectedNamespace}&name=${releaseName}&url=${chartURL}`,
```

Comment:
Shouldn't the param values be encoded? At least `chartURL` since its value comes from an external source.

Reply:
Done